infra/src/main/java/com/redhat/service/bridge/infra/models/dto/ProcessorDTO.java

```diff
@@ -16,19 +16,23 @@
     @JsonProperty("definition")
     private ProcessorDefinition definition;
 
-    @JsonProperty("bridge")
-    private BridgeDTO bridge;
+    @JsonProperty("bridgeId")
+    private String bridgeId;
+
+    @JsonProperty("customerId")
+    private String customerId;
```

Comment:
This is the change in the `ProcessorDTO`: `BridgeDTO` is replaced by `bridgeId` and `customerId`